### DISCUSSION TITLE
Add the bat boss to Karazhan with its cute little bat pet

### DISCUSSION THREAD

**Discussion by siorse** [chromiecraft/chromiecraft#6044] [Suggestions]
Of course, Karazhan is not yet out, so maybe he is already included.

Maybe he isn't.

In the situation where he isn't - could he please be added, so that we may get the cute little bat pet?

Everyone knows a raid becomes 10x more fun when a cute little pet dude is added as a drop.

**Comment by xinmpg** [comment]
/signed
-salamanca

**Comment by Maleyah** [comment]
GIEF! Please? And thank you <3

**Comment by heyitsbench** [comment]
The bat boss in Karazhan is a limited time event during the Wrath pre-patch scourge invasion. I don't believe it's currently set in stone, but currently the possibility for that event to play out on CC [is there](https://github.com/chromiecraft/chromiecraft/discussions/5723). I do not agree with having the bat boss and cute little bat pet be spawned outside of this event.

**Comment by siorse** [comment]
If they do the pre wrath patch event, I don't know if that will happen as the patch we play on is end of wrath

**Reply by heyitsbench** [reply]
I don't see why not, seeing as I don't know of anything that's explicitly been ripped out of the game that was used in that event. Plus they ran the AQ opening event and the TBC pre-patch event.
